scripts/wsrep_sst_mariabackup.sh

```diff
@@ -868,10 +868,10 @@ then
         fi
 
         if [ -n "${WSREP_SST_OPT_PSWD:-}" ]; then
-           INNOEXTRA+=" --password=$WSREP_SST_OPT_PSWD"
+            export MYSQL_PWD=$WSREP_SST_OPT_PSWD
         elif [[ $usrst -eq 1 ]];then
            # Empty password, used for testing, debugging etc.
-           INNOEXTRA+=" --password="
+            export MYSQL_PWD=
```

Comment:
How about `unset` rather than clearing it.

Reply:
As you wish!